admin-frontend/src/components/checkIn.js

```diff
@@ -75,9 +75,15 @@ const CheckInView = (props) => {
         .then(response => {
           const { showNotification } = props;
           if (response.statusCode < 200 || response.statusCode >= 300) {
-            showNotification('Jokin meni pieleen! Kokeile uudestaan.', 'warning')
+              showNotification('Jokin meni pieleen! Kokeile uudestaan.', 'warning')
           } else {
-            handleCheckInSuccess();
+            if (response.success === false) {
```

Comment:
I started to think about the type of the success, and backtracked the response variable. So it's apparently the return promise of fetch API but with the response body parsed as JSON, right? Was just wondering how robust this is :o Maybe I'll ask some more about this :)